engine/datapool-collector/src/main/kotlin/io/holunda/camunda/datapool/DataEntrySenderConfiguration.kt

```diff
@@ -1,31 +1,50 @@
 package io.holunda.camunda.datapool
 
 import io.holunda.camunda.datapool.projector.DataEntryProjector
-import io.holunda.camunda.datapool.sender.SimpleDataEntryCommandSender
-import io.holunda.camunda.datapool.sender.DataEntryCommandSender
+import io.holunda.camunda.datapool.sender.*
 import org.axonframework.commandhandling.gateway.CommandGateway
+import org.slf4j.LoggerFactory
 import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
 import org.springframework.boot.context.properties.EnableConfigurationProperties
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.ComponentScan
 import org.springframework.context.annotation.Configuration
 
 /**
- * Configuration enabled by the property
+ * Configuration sender configuration.
```

Comment:
Comment doesn't make sense this way and seems redundant to me. Reword or remove.